addons/disable-contentportal-detection/bootstrap.js

```diff
@@ -0,0 +1,23 @@
+/* -*- indent-tabs-mode: nil; js-indent-level: 2 -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* exported install, uninstall, startup, shutdown */
+ /* eslint no-implicit-globals: "off" */
+
+const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+
+const {Preferences} = Cu.import("resource://gre/modules/Preferences.jsm", {});
+
+function install() {}
+
+function uninstall() {}
+
+function startup() {
+  let defaults = new Preferences({defaultBranch: true});
+  // ref: http://searchfox.org/mozilla-central/rev/9a7fbdee1d54f99cd548af95b81231d80e5f9ad1/browser/app/profile/firefox.js#628
```

Comment:
Sorry this is a little pedantic, but if you want to link this (which I think is a good idea!) you should probably link to the revision of this file that shipped w/ Firefox 52.0.2 https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_52_0_2_RELEASE/browser/app/profile/firefox.js#l599

I know the line is correct on `mozilla-central` and you permalinked, just might be confusing if someone is looking at this later.